src/cookbook/games/achievements-leaderboard.md

```diff
@@ -0,0 +1,352 @@
+---
+title: Add achievements and leaderboards to your mobile game
+description: >
+  How to use the games_services plugin to add functionality to your game.
+---
+
+Gamers have various motivations for playing games. In broad strokes,
+there are four major motivations: 
+[immersion, achievement, cooperation, and competition](https://meditations.metavert.io/p/game-player-motivations).
+No matter the game you build, some players want to *achieve* in it. This
+could be trophies won or secrets unlocked. Some players want to
+*compete* in it. This could be hitting high scores or accomplishing
+speedruns. These two ideas map to the concepts of *achievements* and
+*leaderboards*.
+
+![A simple graphic representing the four types of motivation explained above](/assets/images/docs/cookbook/types-of-gamer-motivations.png)
+
+Ecosystems such as the App Store and Google Play provide centralized
+services for achievements and leaderboards. Players have achievements
+from all the games in one place and developers don't need to
+re-implement them for every game.
+
+This recipe demonstrates how to use the `games_services` package to add
+achievements and leaderboard functionality to your mobile game.
+
+## 1. Enable platform services
+
+To enable games services, set up *Game Center* on iOS and *Google Play
+Games Services* on Android.
+
+### iOS
```

Comment:
From a brief glance at the [GameKit docs](https://developer.apple.com/documentation/gamekit/), it seems it supports macOS. Would these instructions work for macOS and the Mac App Store too?

If so, might be worth calling this section "iOS and macOS" and updating the various mentions, or at least calling out somewhere in the section that these instruction can be applied to macOS too.

Reply:
I haven't tested this, so I'd rather underpromise than overpromise. I _think_ GameKit behaves a bit differently on macOS than on iOS, so it needs further investigation. I'm making a note to self.

Reply:
Sounds good, expanding things later is great too :)